internal/auth/yaml_storage.go

```diff
@@ -20,6 +23,14 @@ type YamlStorage struct {
 func (y *YamlStorage) Setup(config config.AuthStorage, encryptionPassphrase string) error {
 	y.StorageEncryptionKey = encryptionPassphrase
 	y.StoragePath = config.Config["path"].(string)
+	if strings.HasPrefix(y.StoragePath, "~/") && (runtime.GOOS == "linux" || runtime.GOOS == "darwin") ||
+		strings.HasPrefix(y.StoragePath, "%userprofile%") && runtime.GOOS == "windows" {
+		homeDir, err := os.UserHomeDir()
+		if err != nil {
+			return err
+		}
+		y.StoragePath = filepath.Join(homeDir, y.StoragePath[2:])
```

Comment:
`y.StoragePath[2:]` is wrong on windows. I wonder why not take the easy route and use `"~"+filepath.Separator` everywhere? We'd have to document it, but otherwise it would just work.

Reply:
Sorry for the long absence :see_no_evil: . I have reverted my latest (faulty) commit so it should be working again with `~/`. Only thing left to do is documentation. I could add it to the main README (though it is missing a section for the `auth` block. Maybe worth looking into?) or as a comment in the `example.sync.yaml` file. Or could do both of course. What do you think? @MichaelEischer

Reply:
I'd prefer to document it in both places.